scheme/deltas/041-general-review.sql

```diff
@@ -0,0 +1,27 @@
+DROP INDEX peerServerId ON LcrGateways;
+DROP INDEX name_domain_idx ON Terminals;
+DROP INDEX terminalId ON Users;
+
+ALTER TABLE Users CHANGE active active TINYINT(1) UNSIGNED DEFAULT '0' NOT NULL; 
```

Comment:
What about using MODIFY instead of CHANGE if no renames are made?

ALTER TABLE Users MODIFY active TINYINT(1) UNSIGNED NOT NULL DEFAULT '0'; 

I have also reordered the last part of the query, but that's just a personal obsession (the final result is just the same).